src/routes/(app)/admin/debug/+page.svelte

```diff
@@ -2,56 +2,94 @@
+          method="POST"
+          class="flex items-center gap-4"
+        >
+          <button type="submit" class="btn">Sync with Keycloak</button>
+          <p>This will push mandates and pull email addresses.</p>
+        </form>
+      </section>
+    {/if}
 
-<section>
-  <h1 class="text-lg font-semibold">Metadata</h1>
-  <ul class="ml-4 list-disc">
-    <li>
-      Version:
-      <pre class="inline">{version}</pre>
-      <span
-        class="badge badge-neutral tooltip tooltip-bottom aspect-square px-1"
-        data-tip="This is guaranteed to match the version of the code that is running and is retrieved at build-time. It optionally suffixes the tag name with the number of additional commits on top of the tagged commit (i.e -2) and the abbreviated hash of the most recent commit (i.e -g<hash>)."
-      >
-        ?
-      </span>
-    </li>
-    <li>
-      Prisma log level:
-      <pre class="inline">{data.prismaLogLevel}</pre>
-    </li>
-    <li>Is nollning: {data.isNollning}</li>
-    <li>
-      Is app: {data.isApp}
-    </li>
-    <li>
-      App info: {JSON.stringify(data.appInfo)}
-    </li>
-  </ul>
-</section>
+    <section>
+      <h1 class="text-lg font-semibold">Metadata</h1>
+      <ul class="ml-4 list-disc">
+        <li>
+          Version:
+          <pre class="inline">{version}</pre>
+          <span
+            class="badge badge-neutral tooltip tooltip-bottom aspect-square px-1"
+            data-tip="This is guaranteed to match the version of the code that is running and is retrieved at build-time. It optionally suffixes the tag name with the number of additional commits on top of the tagged commit (i.e -2) and the abbreviated hash of the most recent commit (i.e -g<hash>)."
+          >
+            ?
+          </span>
+        </li>
+        <li>
+          Prisma log level:
+          <pre class="inline">{data.prismaLogLevel}</pre>
+        </li>
+        <li>Is nollning: {data.isNollning}</li>
+        <li>
+          Is app: {data.isApp}
+        </li>
+        <li>
+          App info: {JSON.stringify(data.appInfo)}
+        </li>
+      </ul>
+    </section>
 
-<section class="mt-4">
-  <h1 class="text-lg font-semibold">
-    User: {user.studentId ?? user.externalCode}
-  </h1>
-  <ul class="ml-4 list-disc text-sm">
-    {#each policies as permission}
-      <li>{permission}</li>
-    {/each}
-  </ul>
-</section>
+    <section class="mt-4">
+      <h1 class="text-lg font-semibold">
+        User: {user.studentId ?? user.externalCode}
+      </h1>
+      <ul class="ml-4 list-disc text-sm">
+        {#each policies as permission}
+          <li>{permission}</li>
+        {/each}
+      </ul>
+    </section>
+  </div>
+  <div>
+    <section>
+      <h1 class="text-lg font-semibold">Feature flags</h1>
+      {#each featureFlags as flag}
+        <div class="flex gap-5">
+          {flag}
+          <input
+            type="checkbox"
+            class="toggle"
+            checked={flags.get(flag)}
```

Comment:
The state and the component isn't in sync. If you disable `setDefaultFlags();` (as described in my other comment), the `checked` is not working as intended

Reply:
Good catch!